tests/gutenberg_ro.e2e.spec.js

```diff
@@ -0,0 +1,350 @@
+                // Press enter 2 time to go and visit the first result of the search
+                // I was not able to find a better way to do this feel free to change this
+                // const searchList = await driver.wait(until.elementsLocated(By.xpath('//*[@id="articleList"]/a')), 1500);
+                await driver.sleep(1000);
+                await searchBox.sendKeys(Key.ENTER);
+                await driver.sleep(1000);
+                await searchBox.sendKeys(Key.ENTER);
+
+                await driver.sleep(500);
+                // if title is not loaded in next 4 seconds then return empty string and fail test
+                await searchBox.clear()
+                await driver.switchTo().frame('articleContent');
+                const authorAndBookName = await driver.wait(until.elementLocated(By.id('id00000')), 1500).getText()
+                assert.equal(authorAndBookName, 'MIHAI EMINESCU, POET AL FIINTEI');
+            });
+            it('Navigating back', async function () {
+                // button lies on main page so we need to switch to default content
+                await driver.switchTo().defaultContent();
+                const btnBack = await driver.wait(until.elementLocated(By.id('btnBack')));
+                // I am not sure why i need to click a button two times to go back
+                // Maybe since the first click is to focus on the button and the second one is to click it or element is a <a> not <button>
+                await btnBack.click();
+                await btnBack.click();
+                await btnBack.click();
+                // Title lies in iframe so we need to switch to it
+                await driver.switchTo().frame('articleContent');
+                // in some browsers the title is loaded slowly so we need to wait for it
+                // if title is not loaded in next 4 seconds then return empty string and fail test
+                const mainTitle = await driver.wait(until.elementLocated(By.xpath('//*[@class="main_title"]/h1')), 4000).getText();
+                assert.equal(mainTitle, 'Project Gutenberg Library');
+            });
+            it('Author search Autocomplete', async function () {
+                if (isJqueryMode) {
+                    console.log('\x1b[33m%s\x1b[0m', '      Test skipped.');
+                    return;
+                }
+                const filter = await driver.wait(until.elementIsVisible(driver.findElement(By.id('author_filter'))), 1500)
+                await filter.sendKeys('Mihai Eminescu');
+                const searchList = await driver.wait(until.elementsLocated(By.id('ui-id-1')), 1500);
+                assert.equal(searchList.length, 1);
+            });
+            it('Author search Results', async function () {
+                if (isJqueryMode) {
+                    console.log('\x1b[33m%s\x1b[0m', '      Test skipped.');
+                    return;
+                }
+                // something is wrong here
```

Comment:
Any hint at what is wrong?

Reply:
Sorry for this quiz I forgot to fix this part 
Answer:It was the id of `searchList` 
❌ `until.elementsLocated(By.className('ui-menu-item'))`: get's the actual elements of all the search results
✔ `until.elementsLocated(By.id('ui-id-1'))` : get's the container of all the search results